<launch>
    <node name="left" pkg="sitl_dvrk" type="pub_cam.py"  ns="/vr" output="screen">
        <!-- <param name="cam_type"   type="str"    value="30"/> -->
        <!-- <param name="calib_dir"  type="str"    value="L2R"/> -->
        <!-- <param name="calib_dir"  type="str"    value="R2L"/> -->
        <!-- <param name="calib_type" type="str"    value="matlab"/> -->
        <!-- <param name="calib_type" type="str"    value="opencv"/> -->
        <param name="resolution" type="str"    value="HD1080"/>
        <!-- <param name="calib_path" type="str"    value="/home/$(env USER)/ecm_si_calib_data"/> -->
        <!-- <param name="calib_path" type="str"    value="/home/$(env USER)/ecm_calib_data"/> -->
        <param name="cam_id"     type="int"    value="0"/>
        <param name="gamma"      type="double" value="1.5"/>
        <param name="fps"        type="int"    value="30"/>
        <param name="brightness" type="int"    value="128"/>
        <param name="contrast"   type="int"    value="128"/>
        <param name="saturation" type="int"    value="128"/>
    </node>

    

    <?ignore

    <node name="right" pkg="sitl_dvrk" type="pub_cam.py"  ns="/vr" output="screen">
        <param name="cam_type"   type="str"    value="30"/>
        <param name="calib_dir"  type="str"    value="L2R"/>
        <!-- <param name="calib_dir"  type="str"    value="R2L"/> -->
        <!-- <param name="calib_type" type="str"    value="matlab"/> -->
        <param name="calib_type" type="str"    value="opencv"/>
        <param name="resolution" type="str"    value="HD720"/>
        <param name="calib_path" type="str"    value="/home/$(env USER)/ecm_si_calib_data"/>
        <!-- <param name="calib_path" type="str"    value="/home/$(env USER)/ecm_calib_data"/> -->
        <param name="cam_id"     type="int"    value="4"/>
        <param name="gamma"      type="double" value="1.5"/>
        <param name="fps"        type="int"    value="60"/>
        <param name="brightness" type="int"    value="-11"/>
        <param name="contrast"   type="int"    value="148"/>
        <param name="saturation" type="int"    value="180"/>
    </node>

    <node name="left_rect" pkg="sitl_dvrk" type="pub_cam_rect.py"  ns="/vr" output="screen">
        <param name="cam_type"   type="str"    value="30"/>
        <param name="calib_dir"  type="str"    value="L2R"/>
        <!-- <param name="calib_dir"  type="str"    value="R2L"/> -->
        <!-- <param name="calib_type" type="str"    value="matlab"/> -->
        <param name="calib_type" type="str"    value="opencv"/>
        <param name="resolution" type="str"    value="HD720"/>
        <param name="calib_path" type="str"    value="/home/$(env USER)/ecm_si_calib_data"/>
        <!-- <param name="calib_path" type="str"    value="/home/$(env USER)/ecm_calib_data"/> -->
    </node>

    <node name="right_rect" pkg="sitl_dvrk" type="pub_cam_rect.py"  ns="/vr" output="screen">
        <param name="cam_type"   type="str"    value="30"/>
        <param name="calib_dir"  type="str"    value="L2R"/>
        <!-- <param name="calib_dir"  type="str"    value="R2L"/> -->
        <!-- <param name="calib_type" type="str"    value="matlab"/> -->
        <param name="calib_type" type="str"    value="opencv"/>
        <param name="resolution" type="str"    value="HD720"/>
        <param name="calib_path" type="str"    value="/home/$(env USER)/ecm_si_calib_data"/>
        <!-- <param name="calib_path" type="str"    value="/home/$(env USER)/ecm_calib_data"/> -->
    </node>
    ?>
</launch>